### PR TITLE
Migration of account_invoice_line_description

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please take the ocassion to rename it to _account_invoice_line_description_

**Comment by tafaRU** [comment]
@pedrobaeza done.

File: account_invoice_line_description/invoice.py [hunk 1]
```diff
@@ -19,41 +19,38 @@
...
                 if group_id in user_groups:
-                    product_obj = self.pool.get('product.product')
-                    product = product_obj.browse(
-                        cr, uid, product_id, context=context)
+                    product = self.env['product.product'].browse(product)
                     if (
-                        product
-                        and product.description
-                        and 'value' in res
+                        product and
```
Comment (pedrobaeza) [inline account_invoice_line_description/invoice.py]: You can omit this condition

**Comment by tafaRU** [comment]
@pedrobaeza I addressed your remark in my last commit. Thanks!

File: account_invoice_line_description/invoice.py [hunk 2]
```diff
@@ -19,41 +19,34 @@
...
-class account_invoice_line(orm.Model):
+class AccountInvoiceLine(models.Model):
     _inherit = "account.invoice.line"
 
+    @api.multi
     def product_id_change(
-        self, cr, uid, ids, product_id, uom_id, qty=0,
-        name='', type='out_invoice', partner_id=False, fposition_id=False,
-        price_unit=False, currency_id=False, context=None, company_id=None
+            self, product, uom_id, qty=0, name='', type='out_invoice',
```
Comment (pedrobaeza) [inline account_invoice_line_description/invoice.py:30]: Well spotted the change of the argument names!

**Comment by pedrobaeza** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Wrong button!

**Comment by tafaRU** [comment]
@pedrobaeza :wink:

File: account_invoice_line_description/invoice.py [hunk 3]
```diff
@@ -19,41 +19,34 @@
...
-                'group_use_product_description_per_inv_line'
-            )
-            if ref and len(ref) > 1 and ref[1]:
-                group_id = ref[1]
+            ref = self.env.ref(
+                'invoice_line_description.'
+                'group_use_product_description_per_inv_line')
+            if ref:
+                group_id = ref.id
                 if group_id in user_groups:
```
Comment (guewen) [inline account_invoice_line_description/invoice.py]: Same comment than https://github.com/OCA/account-invoicing/pull/72/files#r29494011

**Comment by tafaRU** [comment]
@guewen I addressed your comment, thanks!

**Comment by eLBati** [comment]
:+1: thanks

**Pull request merged**